README.Rmd

```diff
@@ -814,6 +814,19 @@ ctr <- theophylline(settings = pmx_settings(is.draft = FALSE))
 
 ```
 
+### Reordering facet panels
+
+The order of factors in a facet plot is determined by the data contained in the predictions data frame. Ordering of facet panels can be performed by changing the factor specification for facet columns in the *predictions* data frame.
+
+The example below defines a Controller with changed SEX facet panels order: 1 will be plotted before 0 rather than the other way round.
+
+```{r, eval=FALSE}
+ctr <- theophylline()
```

Comment:
This doesn't work for me.  `a` is not found

Reply:
Addressed by f03cc72f0ded0ce0d42f8b513dc2dc1e84cafae2